oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -955,15 +940,23 @@ impl Runtime {
     }
 
     /// Determine the readable status of a channel, returning:
-    /// - [`ChannelReadStatus::ReadReady`] if there is at least one message in the channel.
-    /// - [`ChannelReadStatus::Orphaned`] if there are no messages and there are no writers
-    /// - [`ChannelReadStatus::NotReady`] if there are no messages but there are some writers
+    /// - [`Ok(ChannelReadStatus::ReadReady)`] if there is at least one message in the channel.
```

Comment:
I think these are not valid links now, are they?

Reply:
You are right. Thanks. I have fixed them now. Although this function is not public, so at the moment, the documentation is not being generated for it.